app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -78,6 +79,10 @@ class MS.Views.TimelineTableView extends Backbone.View
     .fail (response) ->
       MS.alert(response.responseText)
 
+  addDependentStep: (e) ->
+    e.preventDefault()
+    @stepModal.new(@loanId, @refresh.bind(@), {precedingStepId: @stepIdFromEvent(e)})
```

Comment:
Also, in coffeescript as in ruby, you don't need the `{}` around the hash if it's the last argument being passed.